src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -1786,102 +1739,80 @@ void SizePurchasedAir(EnergyPlusData &state, int const PurchAirNum)
                 if (MaxCoolTotCapDes < HVAC::SmallLoad) {
                     MaxCoolTotCapDes = 0.0;
                 }
-                if (IsAutoSize) {
-                    PurchAir(PurchAirNum).MaxCoolTotCap = MaxCoolTotCapDes;
```

Comment:
[src/EnergyPlus/PurchasedAirManager.cc:1789]:(style),[knownConditionTrueFalse],Condition 'IsAutoSize' is always false